### PR TITLE
Allow jupyterhub-githubauth to be hosted on Kubernetes

### PR DESCRIPTION
**Pull request by manics**
Switching back from [running on Jupyterhub on the host](https://github.com/IDR/ansible-role-idr-jupyter/pull/3) to running in docker (via Kubernetes)

### CODE REVIEW

**Comment by joshmoore** [comment]
Propose 0.4.0?

**Comment by manics** [comment]
Yes

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,12 @@
-FROM jupyterhub/jupyterhub:0.6.1
+FROM jupyterhub/jupyterhub:0.7.2
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
 RUN pip install -U pip
 RUN pip install dockerspawner==0.7.0
-RUN pip install https://github.com/IDR/oauthenticator/archive/0.5.1-IDR1.zip
+# TODO: Fork and tag kubespawner (last pypi release is too old)
```
Comment (joshmoore) [inline Dockerfile]: * https://github.com/IDR/kubespawner/releases/tag/0.5.2-IDR1
 * https://github.com/jupyterhub/kubespawner/issues/62

**Comment by manics** [comment]
Updated, though not yet tested.

**Comment by joshmoore** [comment]
Ok. Going to go ahead and tag as 0.4.0, worst case we need a bug fix.

**Pull request merged**